boards/modalai/fc-v2/init/rc.board_sensors

```diff
@@ -22,7 +22,7 @@ icm42688p -s -b 1 -R 12 start
 icm42688p -s -b 2 -R 12 start
 
 # Internal I2C mag
-bmm150 -I start
+bmm150 -I -R 0 start
```

Comment:
0 is already the default (?)

Reply:
This came as a request from @dagar to @modaltb to set an explicit rotation param.

Reply:
@bkueng yes has to do with vendors using different orientations with the same sensor and occasionally the system not able to auto detect and causing issues (e.g. mRo)

Reply:
Minor miscommunication, I was talking about explicitly starting the external mag that you commonly use.

The ist8310 on the Holybro GPS puck (eg m500) is mounted different than the mRo units. Additionally because the Modal AI fc-v1 board doesn't have an internal mag the external rotation won't be fixed automatically at calibration time.